### PR TITLE
Trigger CI for 5472

### PR DESCRIPTION
**Pull request by beeender**
See #5472

### CODE REVIEW

File: examples/build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
-project.ext.sdkVersion = 26
-project.ext.buildTools = '26.0.2'
+project.ext.sdkVersion = 27
+project.ext.minSdkVersion = 15
+project.ext.buildTools = '27.0.0'
```
Comment (zaki50) [inline examples/build.gradle]: `27.0.1` is the latest one.

File: realm.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-gradleVersion=4.2.1
+gradleVersion=4.3
```
Comment (zaki50) [inline realm.properties]: `4.3.1` is the latest version.

File: library-benchmarks/build.gradle [hunk 3]
```diff
@@ -27,12 +27,12 @@ apply plugin: 'com.android.library'
...
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
+    buildToolsVersion "27.0.0"
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 22 // Below 23 to avoid new permission system introduced in M
+        targetSdkVersion 27
```
Comment (zaki50) [inline library-benchmarks/build.gradle]: No need to add a code for runtime permission?

File: realm/realm-library/build.gradle [hunk 4]
```diff
@@ -40,12 +40,12 @@ ext.lcachePath = project.findProperty('lcachePath') ?: System.getenv('NDK_LCACHE
 ext.enableDebugCore = project.hasProperty('enableDebugCore') ? project.getProperty('enableDebugCore') : false //FIXME Use 'false' as default until https://github.com/realm/realm-java/issues/5354 is fixed
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion '26.0.2'
+    compileSdkVersion 27
+    buildToolsVersion '27.0.0'
```
Comment (zaki50) [inline realm/realm-library/build.gradle]: `27.0.1` is the latest.

File: Dockerfile [hunk 5]
```diff
@@ -52,9 +52,9 @@ RUN sdkmanager --update
 # Accept all licenses
 RUN yes y | sdkmanager --licenses
 RUN sdkmanager 'platform-tools'
-RUN sdkmanager 'build-tools;26.0.2'
+RUN sdkmanager 'build-tools;27.0.0'
```
Comment (zaki50) [inline Dockerfile]: `27.0.1` is the latest.

File: README.md [hunk 6]
```diff
@@ -59,10 +66,10 @@ In case you don't want to use the precompiled version, you can build Realm yours
 
 ### Prerequisites
 
- * Download the [**JDK 7**](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html) or [**JDK 8**](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) from Oracle and install it.
- * Download & install the Android SDK **Build-Tools 26.0.2**, **Android O (API 26)** (for example through Android Studio’s **Android SDK Manager**).
+ * Download the [**JDK 8**](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) from Oracle and install it.
+ * Download & install the Android SDK **Build-Tools 27.0.0**, **Android Oreo (API 27)** (for example through Android Studio’s **Android SDK Manager**).
```
Comment (zaki50) [inline README.md]: `27.0.1` is the latest build-tools.

File: examples/newsreaderExample/build.gradle [hunk 7]
```diff
@@ -39,16 +39,16 @@ android {
 
 dependencies {
     //noinspection GradleDependency
-    implementation 'com.android.support:appcompat-v7:26.0.1'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
```
Comment (zaki50) [inline examples/newsreaderExample/build.gradle]: `27.0.1` is the latest support library.

File: examples/newsreaderExample/build.gradle [hunk 8]
```diff
@@ -39,16 +39,16 @@ android {
...
     //noinspection GradleDependency
-    implementation 'com.android.support:design:26.0.1'
-    implementation 'com.jakewharton.timber:timber:4.1.0'
+    implementation 'com.android.support:design:27.0.0'
```
Comment (zaki50) [inline examples/newsreaderExample/build.gradle]: `27.0.1` is the latest support library.

File: examples/objectServerExample/build.gradle [hunk 9]
```diff
@@ -57,10 +57,9 @@ realm {
 }
 
 dependencies {
-    implementation 'com.android.support:support-v4:26.0.1'
-    implementation 'com.android.support:appcompat-v7:26.0.1'
-    implementation 'com.android.support:design:26.0.1'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
+    implementation 'com.android.support:design:27.0.0'
```
Comment (zaki50) [inline examples/objectServerExample/build.gradle]: `27.0.1` is the latest support library.

File: examples/rxJavaExample/build.gradle [hunk 10]
```diff
@@ -37,7 +37,7 @@ android {
 dependencies {
     implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
     implementation 'io.reactivex.rxjava2:rxjava:2.1.0'
-    implementation 'com.android.support:appcompat-v7:26.0.1'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
```
Comment (zaki50) [inline examples/rxJavaExample/build.gradle]: `27.0.1` is the latest support library.

File: examples/secureTokenAndroidKeyStore/build.gradle [hunk 11]
```diff
@@ -30,10 +30,10 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestImplementation('com.android.support.test.espresso:espresso-core:3.0.1', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    implementation 'com.android.support:appcompat-v7:26.0.1'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
```
Comment (zaki50) [inline examples/secureTokenAndroidKeyStore/build.gradle]: `27.0.1` is the latest support library.

File: examples/threadExample/build.gradle [hunk 12]
```diff
@@ -25,5 +25,5 @@ android {
 
 dependencies {
     //noinspection GradleDependency
-    implementation 'com.android.support:appcompat-v7:24.0.0'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
```
Comment (zaki50) [inline examples/threadExample/build.gradle]: `27.0.1` is the latest support library.

File: examples/unitTestExample/build.gradle [hunk 13]
```diff
@@ -34,9 +34,9 @@ android {
 
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:26.0.1'
+    implementation 'com.android.support:appcompat-v7:27.0.0'
```
Comment (zaki50) [inline examples/unitTestExample/build.gradle]: `27.0.1` is the latest support library.

File: library-benchmarks/build.gradle [hunk 14]
```diff
@@ -27,12 +27,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'realm-android'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
+    buildToolsVersion "27.0.0"
```
Comment (zaki50) [inline library-benchmarks/build.gradle]: `27.0.1` is the latest.

File: CHANGELOG.md [hunk 15]
```diff
@@ -10,10 +10,12 @@
 ### Internal
 
 * Use `OsList` instead of `OsResults` to add notification token on for `RealmList<RealmModel>`.
+* Updated gralde and plugins to support Android Studio `3.0.0` (#5472).
```
Comment (cmelchior) [inline CHANGELOG.md]: Gradle

File: CHANGELOG.md [hunk 16]
```diff
@@ -10,10 +10,12 @@
...
 
 ### Credits
 
 * Thanks to @tbsandee for fixing a typo (#5548).
+* Thanks to @vivekkiran for updating gralde and plugins to support Android Studio `3.0.0` (#5472).
```
Comment (cmelchior) [inline CHANGELOG.md]: Gradle

File: library-benchmarks/build.gradle (hunk 3 above)
Comment (beeender) [reply library-benchmarks/build.gradle]: Tried, it doesn't ask for runtime permissions. Not sure why the comment was there.

**Pull request merged**